docs/library/graphing/piechart.md

```diff
@@ -149,40 +182,33 @@ class PieChartState(rx.State):
                 break
 
 
-pie_chart_state_example = """
-rx.hstack(
-    rx.recharts.pie_chart(
-        rx.recharts.pie(
-            data=PieChartState.resources,
-            data_key="count",
-            name_key="type_",
-            cx="50%",
-            cy="50%",
-            start_angle=180,
-            end_angle=0,
-            fill="#8884d8",
-            label=True,
+def dynamic_pie_example():
```

Comment:
This example should have labels or different colors because I can't tell which data corresponds to each quadrant

Reply:
doesn't seem to be a way to do this with the props we currently have